File: 은지/LV2/모음 사전.py
def solution(word):
    answer = 0
    info = {"E": 1, "I": 2, "O": 3, "U": 4}
    for i in range(len(word)):
        if word[i] == "A":
            answer += 1
        else:
            answer += info[word[i]]+1
            for j in range(i+1, 5):
                answer += (5**(5-j))*info[word[i]]

    return answer

File: 은지/LV2/모음 사전.js
function solution(word) {
  let answer = 0;
  let info = ["A", "E", "I", "O", "U"]

  for (let i=0; i<word.length; i++) {
    if (word[i] === "A") {
      answer += 1
    } 
    else {
      let idx = info.indexOf(word[i])
      answer += idx + 1
      for (let j=i+1; j<5; j++) {
        answer += (5 ** (5-j)) * idx
      }
    }
  }

  return answer;
}